wooting-analog-plugin/src/lib.rs

```diff
@@ -47,7 +47,7 @@ trait DeviceImplementation: objekt::Clone + Send {
             device.product_id()
         };
         //Check if the pid & hid match
-        pid.eq(&hid.pid)
+        (hid.pid == 0 || pid.eq(&hid.pid))
```

Comment:
It would be nicer to make pid `Option<u16>` to handle the undefined state, rather than 0